conda.recipe/meta.yaml

```diff
@@ -15,31 +15,40 @@ source:
 requirements:
   build:
```

Comment:
That's a good question -- I'll investigate.  I knew at one point, but I've since forgotten.  It would be good to whittle this list down to a minimum.